src/main/java/ru/svgogin/service/spark/handler/RestResponseEntityExceptionHandler.java

```diff
@@ -20,19 +31,66 @@
 
   @ExceptionHandler(value = {EntityAlreadyExistsException.class})
   protected ResponseEntity<Object> handleConflict(RuntimeException ex, WebRequest request) {
-    String message = ex.getMessage();
+    var message = ex.getMessage();
+    var errorCode = ErrorCode.ERROR001;
+    var bodyOfResponse = List.of(new ErrorDto(errorCode, message));
     log.warn(message);
-    var bodyOfResponse = new ErrorDto(ErrorDto.ErrorCode.ERROR001, message);
     return handleExceptionInternal(ex, bodyOfResponse,
         new HttpHeaders(), HttpStatus.CONFLICT, request);
   }
 
   @ExceptionHandler(value = {NoSuchEntityException.class})
   protected ResponseEntity<Object> handleNotFound(RuntimeException ex, WebRequest request) {
-    String message = ex.getMessage();
+    var message = ex.getMessage();
+    var errorCode = ErrorCode.ERROR002;
+    var bodyOfResponse = List.of(new ErrorDto(errorCode, message));
```

Comment:
nit: you can inline variables:
```suggestion
    var bodyOfResponse = List.of(new ErrorDto(ErrorCode.ERROR002, ex.getMessage()));
```